lms/static/js/spec/dashboard/dropdown_spec.js

```diff
@@ -0,0 +1,87 @@
+define(['js/dashboard/dropdown', 'jquery.simulate'],
+    function() {
+        'use strict';
+        var keys = $.simulate.keyCode,
+            toggleButtonSelector = '#actions-dropdown-link-2',
+            dropdownSelector = '#actions-dropdown-2',
+            dropdownItemSelector = '#actions-dropdown-2 li a',
+            clickToggleButton = function() {
+                $(toggleButtonSelector).click();
+            },
+            verifyDropdownVisible = function() {
+                expect($(dropdownSelector)).toBeVisible();
+            },
+            verifyDropdownNotVisible = function() {
+                expect($(dropdownSelector)).not.toBeVisible();
+            },
+            waitForElementToBeFocused = function(element, desc) {
```

Comment:
@andy-armstrong @cahrens   Are the JS tests related to focus that are flaky using any sort of `waitsFor` method like this?  I know when I tried to use the `expect().toBeFocused` method, it was flaky.